texticle.gemspec

```diff
@@ -50,5 +50,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'pry'
   s.add_development_dependency 'pry-doc'
 
-  s.add_dependency('activerecord', '~> 3.0')
+  s.add_dependency('activerecord', '~> 4.2.8')
```

Comment:
maybe it's better to point to range, this should work with any ~> 4.0 version I guess. Tests pass in gem?